test/unit/mem/fence/src/FenceTest.cpp

```diff
@@ -14,7 +14,86 @@
 
 #include <catch2/catch.hpp>
 
-class DeviceFenceTestKernel
+// Trait to detect whether an accelerator supports or not multiple threads per block
+template<typename TAcc>
+struct IsSingleThreaded : public std::false_type
+{
+};
```

Comment:
I would just make this a variable template directly and then specialize it later:
```suggestion
template<typename TAcc>
inline constexpr bool isSingleThreaded = false;
```

Reply:
I see... I would prefer to make this change globally, also in the other places that use template variables.